visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -104,159 +123,241 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
+                formattedMinDisparity,
+                formattedMaxDisparity,
+            );
+
+            const insights3 = localization.formatString(
+                localization.ModelComparison.insightsText3,
+                selectedMetric.title.toLowerCase(),
+                selectedMetric.isMinimization ? formattedMinAccuracy : formattedMaxAccuracy,
+                FormatMetrics.formatNumbers(
+                    this.state.disparityArray[selectedMetric.isMinimization ? minAccuracyIndex : maxAccuracyIndex],
+                    this.props.accuracyPickerProps.selectedAccuracyKey,
+                ),
+            );
+
+            const insights4 = localization.formatString(
+                localization.ModelComparison.insightsText4,
+                selectedMetric.title.toLowerCase(),
+                FormatMetrics.formatNumbers(
+                    this.state.accuracyArray[minDisparityIndex],
+                    this.props.accuracyPickerProps.selectedAccuracyKey,
+                ),
+                formattedMinDisparity,
+            );
+
+            const howToReadText = localization.formatString(
+                localization.ModelComparison.howToReadText,
+                this.props.modelCount.toString(),
+                selectedMetric.title.toLowerCase(),
+                selectedMetric.isMinimization
+                    ? localization.ModelComparison.lower
+                    : localization.ModelComparison.higher,
+            );
+
+            const props = _.cloneDeep(this.plotlyProps);
+            props.data = ChartBuilder.buildPlotlySeries(props.data[0], data).map((series) => {
+                series.name = this.props.dashboardContext.modelNames[series.name];
+                series.text = this.props.dashboardContext.modelNames;
+                return series;
+            });
+
+            const accuracyMetricTitle = selectedMetric.title;
+            const parityMetricTitle = ParityOptions[this.props.parityPickerProps.selectedParityKey].title;
+            props.layout.xaxis.title = accuracyMetricTitle;
+            props.layout.yaxis.title = parityMetricTitle;
+
+            const InsightsIcon = () => <Icon iconName="CRMCustomerInsightsApp" className={styles.insightsIcon} />;
+            const DownloadIcon = () => <Icon iconName="Download" className={styles.downloadIcon} />;
+
+            const cancelIcon: IIconProps = { iconName: 'Cancel' };
+
+            mainChart = (
                 <div className={styles.main}>
-                    <div className={styles.chart}>
-                        <AccessibleChart
-                            plotlyProps={props}
-                            sharedSelectionContext={this.props.selections}
-                            theme={undefined}
-                        />
+                    <div className={styles.mainLeft}>
+                        <div className={styles.howTo}>
+                            <Modal
+                                titleAriaId="intro modal"
```

Comment:
just an issue we've recently run into with modals in jupyter, the jupyter environment listens to hotkey presses when focus is in a new layer. So maybe test that yuo can press 'h' or 's' when the modal is open without triggering jupyter controls. Not a big deal if there are no form components in the modal.